src/draw/DefaultDrawer.js

```diff
@@ -46,6 +32,49 @@ class DefaultDrawer {
      * @type {Object}
      */
     this.resources = resources;
+
+    /**
+     * Render svg viewbox width
+     * @type {number}
+     */
+    this.width = options.width || 1280;
+
+    /**
+     * Render svg viewbox height
+     * @type {number}
+     */
+    this.height = options.height || 1280;
+
+    /**
+     * Dictionary of subjects to send events to.
+     * @type {Object}
+     */
+    this.events = events
+
+    /**
+     * Minimum height of a component
+     * @type {number}
+     */
+    this.baseHeight = options.baseHeight || 50;
+
+    /**
+     * Padding around components
+     * @type {number}
+     */
+    this.basePadding = options.basePadding || 30;
+
+    /**
+     * Minimum width of a component
+     * @type {number}
+     */
+    this.baseWidth = options.baseWidth || 230;
+
+    /**
+     * Maximum aggregated value that each line at a give depth can contain
```

Comment:
value of what ? number of character ?

Reply:
Updated associated function behavior to now simply use the number of component per line.
Previous behavior was trying to use the d3 defined value, a number representing the weight of each branch.